### PR TITLE
traceContextInjection init parameter documentation

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do? What is the motivation?

Related to the implementation of [the short term solution in APM <> RUM correlation improvement](https://docs.google.com/document/d/1w7qe6Jp9vF6HmRA5bNjzZI9JxmXq8cwscqpUPpF686Y/edit#heading=h.y8mrkhm00858)

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by cswatt** [comment]
Created DOCS-7721 to track.

File: content/en/real_user_monitoring/platform/connect_rum_and_traces.md [hunk 1]
```diff
@@ -114,9 +114,21 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
 
 **Note**: `traceSampleRate` **does not** impact RUM sessions sampling. Only backend traces are sampled out.
 
+4. _(Optional)_ If you set a `traceSampleRate`, to ensure backend services' sampling decisions are still applied, configure the `traceContextInjection` initialization parameter to `sampled` (set to `all` by default).
+
+_Example_: If you set the `traceSampleRate` to 20% in the Browser SDK:
```
Comment (rtrieu) [inline content/en/real_user_monitoring/platform/connect_rum_and_traces.md:119]:
```suggestion
    For example, if you set the `traceSampleRate` to 20% in the Browser SDK:
```

File: content/en/real_user_monitoring/platform/connect_rum_and_traces.md [hunk 2]
```diff
@@ -114,9 +114,21 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+- When `traceContextInjection` is set to `all`, **20%** of backend traces are kept and **80%** of backend traces are dropped.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/platform/connect_rum_and_traces.md:120]:
```suggestion
    - When `traceContextInjection` is set to `all`, **20%** of backend traces are kept and **80%** of backend traces are dropped.
```

File: content/en/real_user_monitoring/platform/connect_rum_and_traces.md [hunk 3]
```diff
@@ -114,9 +114,21 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+
+{{< img src="real_user_monitoring/connect_rum_and_traces/traceContextInjection_all.png" alt="traceContextInjection set to all" style="width:90%;">}}
```
Comment (rtrieu) [inline content/en/real_user_monitoring/platform/connect_rum_and_traces.md:122]:
```suggestion
  {{< img src="real_user_monitoring/connect_rum_and_traces/traceContextInjection_all.png" alt="traceContextInjection set to all" style="width:90%;">}}
```

File: content/en/real_user_monitoring/platform/connect_rum_and_traces.md [hunk 4]
```diff
@@ -114,9 +114,21 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+
+- When `traceContextInjection` is set to `sampled`, **20%** of backend traces are kept. For the remaining **80%**, the browser SDK **does not inject** a sampling decision. The decision is made on the server side, based on the tracing library head-based sampling [configuration][2] (backend sample rate set to 40% in the example below).
```
Comment (rtrieu) [inline content/en/real_user_monitoring/platform/connect_rum_and_traces.md:124]:
```suggestion
  - When `traceContextInjection` is set to `sampled`, **20%** of backend traces are kept. For the remaining **80%**, the browser SDK **does not inject** a sampling decision. The decision is made on the server side and is based on the tracing library head-based sampling [configuration][2]. In the example below, the backend sample rate is set to 40%, and therefore 32% of the remaining backend traces are kept.
```

File: content/en/real_user_monitoring/platform/connect_rum_and_traces.md [hunk 5]
```diff
@@ -114,9 +114,21 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+
+{{< img src="real_user_monitoring/connect_rum_and_traces/traceContextInjection_sampled.png" alt="traceContextInjection set to sampled" style="width:90%;">}}
```
Comment (rtrieu) [inline content/en/real_user_monitoring/platform/connect_rum_and_traces.md:126]:
```suggestion
    {{< img src="real_user_monitoring/connect_rum_and_traces/traceContextInjection_sampled.png" alt="traceContextInjection set to sampled" style="width:90%;">}}
```

File: content/en/real_user_monitoring/platform/connect_rum_and_traces.md (hunk 4 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/platform/connect_rum_and_traces.md:124]: Hi @dussault-antoine , I added the bit at the end to clarify the example further, but not sure if it is accurate. Can you verify and edit if needed? Thank you!

**Review by rtrieu — CHANGES_REQUESTED** [review]
I've left some feedback for your review.

**Comment by dussault-antoine** [comment]
@rtrieu committed the suggestions ! Thanks ! 
Good to merge

**Comment by rtrieu** [comment]
Thanks @dussault-antoine , letting you know that I've put in a request to Brand Design to make brand-approved versions of your diagrams, so don't be surprised if they don't look different in another update :)

**Review by rtrieu — APPROVED** [review]
Looks good, thanks!

**Pull request merged**